contracts/interfaces/IExtraReward.sol

```diff
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: MIT
 pragma solidity 0.8.13;
-import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/token/ERC20/IERC20.sol";
```

Comment:
any way for mappings to avoid removing the contracts path?

Reply:
Yes, I thought it looked cleaner without it, but I can avoid it all entirely with an extra config.

Reply:
Fixed in https://github.com/yearn/veYFI/commit/095c9b0d1ec54e8c82ae8b7c7da2de19925a5d62#diff-c0795f9aba72a67202997254e1eae063ac93bf18ceb55e6b6175844cf6f73cc8R14